src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { id, title, completed } = todo;
+  const [isEditing, setIsEditing] = useState<boolean>(false);
+  const handleDoubleClick = () => {
+    setIsEditing(true);
+  };
+
+  return (
+    <li
+      className={classNames('todo', {
+        completed,
+        editing: isEditing,
+      })}
+      key={id}
```

Comment:
The 'key' prop is unnecessary on this 'li' element. 'key' is special in React and should be provided to the elements inside a list to give the elements a stable identity.